Workbooks/CosmosDb/Overview/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"Cosmos Db Overview",
+    "name":"Azure Cosmos Db Overview",
```

Comment:
some of the places are `Azure Cosmos DB` with uppercase db, some are `Db`? should they all be the same?

Reply:
yes they should be consistent